packages/editor-ui/src/mixins/nodeHelpers.ts

```diff
@@ -475,11 +476,10 @@ export const nodeHelpers = defineComponent({
 					}
 
 					if (nameMatches.length === 0) {
-						const isInstanceOwner = this.usersStore.isInstanceOwner;
 						const isCredentialUsedInWorkflow =
 							this.workflowsStore.usedCredentials?.[selectedCredentials.id as string];
 
-						if (!isCredentialUsedInWorkflow && !isInstanceOwner) {
+						if (!isCredentialUsedInWorkflow && !this.rbacStore.hasScope('credential:read')) {
```

Comment:
Can you use `hasPermission(['rbac'], { rbac: { scope: 'credential:read' } })` to keep consistent?